data/remnant/remnant missions.txt

```diff
@@ -4479,3 +4479,140 @@ mission "Remnant: Expanded Horizons Astral job"
+	gun 0 0
+
+
+
+mission "Remnant: Expanded Horizons Storms 1"
+	name "Ember Waste Storm Scanning"
+	description "Dawn has asked you to go scan one of the Ember Waste's storms."
+	passengers 1
+	source
+		government "Remnant"
+	waypoint
+		system "Aescolanus"
+	to offer
+		has "Remnant: Expanded Horizons Astral 2: done"
+		random < 30
+	on offer
+		require "Research Laboratory"
+		conversation
+			`As you cross the starport you are suddenly ambushed by Dawn appearing out of a side hallway. "Captain <first>! I have been hoping to catch you. I need some help." She hurries up, waving a datapad full of charts. "I have some additional leads for my research project, and I need a pilot who can take me out to Aescolanus." She looks at you hopefully.`
+			choice
+				`	"Sure, I am always willing to help."`
+				`	"Not today, sorry."`
+					decline
+			`	"Excellent! You already have a research laboratory onboard, and it has all the equipment I need. Now, before we head out, you should know that the goal of this mission is to do some analysis on the leading edge of a storm that will be breaking on the system shortly. We will not have to stay for the whole storm, but we will have to be there for at least the first part."`
+			choice
+				`	"Are there any special adjustments needed?"`
+				`	"The <ship> is ready to go."`
+					goto readyforstorms
+			`	"Well, I expect you are already setup to handle the normal Ember Waste heat, correct?" She continues without waiting for you to reply. "The Ember Waste storms primarily are a risk to electrical systems - typically by draining your power. So you can compensate by ensuring that you have some excess available to use. I know some captains carry extra generators to ensure they always have extra power, but others instead carry extra crystal capacitors so they can outlast the storm. In theory, you could just accept that the your ship will be shut down during the height of the storm, and reboot everything afterwards. So long as no threats are present, it is safe but somewhat nervewracking. Which strategy fits the <ship> is up to you."`
+			label readyforstorms
+			`	"I will settle into a bunk to do some pre-calculations. Please let me know when we arrive in Aescolanus."`
+				accept
+	on waypoint
+		dialog `Dawn quickly sets to work expanding the laboratory and setting up her instruments. After a few minutes she returns to the cockpit and settles into a chair by the window. "Everything is ready, Captain. Now we just wait for the storm to arrive. I will let you know when we are ready to leave."`
+	npc kill
+		government "Test Dummy"
+		system "Aescolanus"
+		ship "_Ion Timer Ship" "Timer"
+		personality unconstrained heroic nemesis
+		dialog `As the storm starts to swell around you the instrumention chimes to indicate that it has collected all the data it needs. Dawn quickly begins the shutdown routine to protect the equipment against potential damage, and gives you the OK to return to <destination>.`
```

Comment:
```suggestion
		dialog `As the storm starts to swell around you, the instrumentation chimes to indicate that it has collected all the data it needs. Dawn quickly begins the shutdown routine to protect the equipment against potential damage, and gives you the OK to return to <planet>.`
```
Never heard of `instrumention` before, and a quick search didn't seem to shed any light on it. I assumed it was a mispelling of `instrumentation`, but let me know if it's scientific jargon or some other term I just don't recognize. I also just though `planet` fit better than `destination` here, but that's personal preference.